regression/pages/common/api_clients.py

```diff
@@ -0,0 +1,260 @@
+"""
+API classes for different tasks
+"""
+import Cookie
+import json
+
+import requests
+from requests.auth import AuthBase
+from edx_rest_api_client.client import EdxRestApiClient
+
+from regression.pages.whitelabel.const import (
+    ACCESS_TOKEN,
+    AUTH_PASSWORD,
+    AUTH_USERNAME,
+    ECOMMERCE_API_URL,
+    ENROLLMENT_API_URL,
+    URL_WITHOUT_AUTH
+)
+
+
+class ApiException(Exception):
+    """
+    Exceptions for API failures
+    """
+    pass
+
+
+def check_response(response):
+    """
+    Check whether a response was successful. If not, raise an exception
+    Args:
+        response:
+    """
+    if response.status_code != 200:
+        raise ApiException(
+            'API request failed with following error code: ' +
+            str(response.status_code)
+        )
+
+
+class BearerAuth(AuthBase):
+    """
+    Attaches Bearer Authentication to the given Request object.
+    """
+
+    def __init__(self, token):
+        """ Instantiate the auth class. """
+        self.token = token
+
+    def __call__(self, r):
+        """ Update the request headers. """
+        r.headers['Authorization'] = 'Bearer {}'.format(self.token)
```

Comment:
I seem to remember that we don't have pylint on this repo? One-letter variables are not the best...could you please re-name?